client/my-sites/email/inbox/mailbox-selection-list/progress-line-style.scss

```diff
@@ -0,0 +1,89 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.progress-line__container {
+	display: flex;
+	flex-direction: column;
+	margin-top: 35vh;
+
+	@include break-large {
+		transform: none;
+	}
+
+
+	.progress-line__status-text-container {
+		align-items: center;
+		display: flex;
+		justify-content: center;
+		margin-bottom: 10px;
+
+		@include break-large {
+			margin-bottom: 20px;
+		}
+
+		.progress-line__status-text {
+			font-family: $sans;
+			font-size: $font-title-small;
+			font-weight: 400;
+			line-height: 22px;
+
+			@include break-large {
+				font-size: $font-title-medium;
+				line-height: 28px;
+			}
+		}
+	}
+
+	.progress-line__loading-container {
+		align-items: center;
+		display: flex;
+		justify-content: center;
+
+		@keyframes load {
+			0% {
+				background: var( --studio-gray-70 );
+				width: 0;
+			}
+
+			25% {
+				background: var( --studio-gray-70 );
+				width: 25%;
+			}
+
+			50% {
+				background: var( --studio-gray-70 );
+				width: 50%;
+			}
+
+			75% {
+				background: var( --studio-gray-70 );
+				width: 75%;
+			}
+
+			100% {
+				background: var( --studio-gray-70 );
+				width: 100%;
+			}
+		}
+
+		.progress-line__loading-loader {
+			border-radius: 2px;
+			box-shadow: inset 0 0 5px rgba( 0, 0, 0, 0.2 );
+			display: flex;
+			height: 4px;
+			min-width: 280px;
+			overflow: hidden;
+			width: 50%;
+
+			@include break-large {
+				max-width: 400px;
+			}
+
+			&:after { /* stylelint-disable-line selector-pseudo-element-colon-notation */
+				animation: load 5s infinite;
+				box-shadow: 0 0 5px rgba( 0, 0, 0, 0.2 );
+				content: '';
```

Comment:
I think it would be much better to end the animation on the last keyframe and do not repeat it forever, as seems to me not interesting for the user, to check that the progress bar is reloading, it might lead to making the user think that something is failing. What do you think @saygunnyc?

```suggestion
				animation: load 5s;
				animation-iteration-count: 1;
				animation-fill-mode: forwards;
				box-shadow: 0 0 5px rgba( 0, 0, 0, 0.2 );
				content: '';
```

Infinite animation loop | static ending
------------ | -------------
![infinite-loop-loading](https://user-images.githubusercontent.com/5689927/134644488-5a0575d4-7b22-4ffb-8bf9-a02a35cc3085.gif) | ![image](https://user-images.githubusercontent.com/5689927/134643865-440cc166-c427-4a40-b643-3ba6486d7ef6.png)

Reply:
💯